ios/Podfile.lock

```diff
@@ -791,4 +791,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: cb0b1503fafdfd47c00196486ddb9af228e8c1da
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.10.1
```

Comment:
I think we need to keep this 1.9.3 for app center?

Reply:
Good catch. I don't know why my local cocoa pods install keeps auto-updating.